pkg/volume/storageos/OWNERS

```diff
@@ -1,6 +0,0 @@
-# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
The maintainers keyword is invalid and all 3 of the people referenced are not org members. Both croomes and chira001 both work for storageos; should their reference still be kept? Maybe a comment in an empty OWNERS file?

Reply:
> Maybe a comment in an empty OWNERS file?

That seems reasonable for now.

@croomes @chira001 let's get you added to the Kubernetes org and then re-added here as approvers for the plugin.